src/poliastro/ground_station.py

```diff
@@ -0,0 +1,145 @@
+import numpy as np
+from astropy import time, units as u
+
+
+class GroundStation(object):
+    """ Class representing a ground station on an arbitrary ellipsoid.
+    """
+
+    def __init__(self, lon, lat, h, epoch, angular_velocity, a, b, c=None):
+        """
+
+        Parameters
+        ----------
+        lon: ~astropy.units.quantity.Quantity
+            geodetic longitude
+
+        lat: ~astropy.units.quantity.Quantity
+            geodetic latitude
+
+        h: ~astropy.units.quantity.Quantity
+            geodetic height
+
+        epoch: ~astropy.units.quantity.Quantity
+            starting epoch
+
+        angular_velocity: ~astropy.units.quantity.Quantity
+            angular velocity
+
+        a: ~astropy.units.quantity.Quantity
+            semi-major axis
+
+        b: ~astropy.units.quantity.Quantity
+            semi-minor axis
+
+        c: ~astropy.units.quantity.Quantity
+            Third axis defining the ellipsoid.
+            If not set, it defaults to the value of a, making the ellipsoid
+            a spheroid.
+        """
+        self._lon = lon
+        self._lat = lat
+        self._h = h
+        self._epoch = epoch
```

Comment:
I guess the `epoch` is only needed for the propagation? As per my remarks about `.propagate`, probably we won't need to specify the *time* of a ground station, only its position in planet-fixed coordinates.